dallinger/experiment_server/experiment_server.py

```diff
@@ -53,6 +47,25 @@
 app = Flask("Experiment_Server")
 
 
+@app.before_request
+def before_request():
+    flask_app_globals.start = time.time()
+
+
+@app.after_request
+def after_request(response):
+    diff = time.time() - flask_app_globals.start
+    loggable_response = (
+        response.response
+        and 200 <= response.status_code < 300
+        and response.content_type.startswith("text/html")
+    )
+    if loggable_response:
+        log = f"[{datetime.now().strftime('%Y-%m-%d %H:%M:%S')}] \"{request.method} {request.path}\" {response.status_code} {response.content_length} {diff}"
+        app.logger.info(log)
```

Comment:
Doing it this way means that we end up with duplicate logs (i.e. the response is logged twice, once with the original Werkzeug logic and once with your logic). Maybe it's better to override the original logger, like this? https://stackoverflow.com/a/36302219/8454486

Reply:
After a discussion with @pmcharrison, we decided the proposed solution is more difficult to implement than expected. I therefore changed the logging message to not look like `werkzeug` logging message.